src/libaktualizr/storage/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 add_custom_command(OUTPUT sql_schemas.cc sql_schemas_target
-    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/embed_schemas.py ${PROJECT_SOURCE_DIR}/config/sql/ ${CMAKE_CURRENT_BINARY_DIR}/sql_schemas.cc
+    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/embed_schemas.py ${PROJECT_SOURCE_DIR}/config/sql/ ${CMAKE_CURRENT_BINARY_DIR}/sql_schemas.cc libaktualizr_
```

Comment:
This is really subjective aesthetics but I don't think it makes sense to not have `_` in the prefix right now, so it would be a bit more elegant to remove it from the cli.

e.g: `embed_schemas.py ... ... ... libaktualiz` and have the python script add the `_` between the two parts itself

Reply:
Yes, you're right, fixed.